lib/nrf_modem_lib/shell/modem_trace_memfault.c

```diff
@@ -61,10 +66,13 @@ static sMemfaultCdrSourceImpl s_my_custom_data_recording_source = {
 	.mark_cdr_read_cb = mark_cdr_read_cb,
 };
 
-void modem_trace_memfault_send(uint32_t trace_duration_ms)
+void modem_trace_memfault_send(const struct shell *sh, size_t size, uint32_t trace_duration_ms)
 {
 	static bool memfault_cdr_source_registered;
-	size_t size = nrf_modem_lib_trace_data_size();
+
+	if (shl == NULL) {
```

Comment:
I guess this NULL check is kind of unnecessary and shl could be set always but doesn't harm either.